vertx-db2-client/src/main/java/io/vertx/db2client/impl/drda/ColumnMetaData.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2019,2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.vertx.db2client.impl.drda;
+
+public class ColumnMetaData {
+    
+    public int columns_;
```

Comment:
why naming with trailing underscores ?

Reply:
this is how the code was from Apache Derby, I'm trying to modify the code as little as possible for starters while we try to get everything working so that it's easier to diff the original code. After we get everything working well I plan to come back to the drda package and do some major refactoring